index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <title>Developer's Summit</title>
+  </head>
+  <body>
+    <!--Navbar-->
+    <nav>
+      <div class="logo-container">
+        <a href="index.html" class="logo">
+          <span class="highlight">Dev</span>Summit<span class="highlight"
+            >.</span
+          >
+        </a>
+      </div>
+      <div class="nav-options">
+        <ul class="menu">
+          <li class="menu-item"><a href="about.html">About</a></li>
+          <li class="menu-item"><a href="#">Program</a></li>
+          <li class="menu-item"><a href="#">Join</a></li>
+          <li class="menu-item"><a href="#">Sponsor</a></li>
+          <li class="menu-item"><a href="#">News</a></li>
+        </ul>
+        <a href="#" class="register-btn"> Get Tickets </a>
+      </div>
+      <!--Toggle menu screen-->
```

Comment:
- [ ] great job on the mobile version. kindly put your mobile hamburger on the left like on the Behance design
your design
![Screenshot (171)_LI](https://user-images.githubusercontent.com/65298048/174777164-7b58515a-56c3-46ea-995a-7f596fa220cf.jpg)
behance 👇🏾 
![Screenshot (169)_LI](https://user-images.githubusercontent.com/65298048/174776990-ca5fa219-714b-4727-adce-f76f9b88ad30.jpg)